ibis/backends/tests/test_generic.py

```diff
@@ -1501,10 +1501,13 @@ def test_distinct_on_keep_is_none(backend, on):
         "trino",  # checksum returns varbinary
     ]
 )
-def test_hash(backend, alltypes):
+@pytest.mark.parametrize(
+    "dtype", ["smallint", "int", "bigint", "float", "double", "string"]
```

Comment:
I'm just not testing `macaddr` here, since `alltypes` doesn't have that column. Should I just add a `macaddr` column?

On top of that, I also think something is off with `macaddr` to begin with, though, since `PostgresType.from_ibis(dt.macaddr)` returns `DataType(this=Type.VARCHAR)`, and `test_macaddr_literal` in ibis/backends/tests/test_network.py expects a text type to be returned; I think the change should be bigger, but wanted to get an opinion first. :)

Reply:
adding a column to `alltypes` will be a headache -- I think we could add a standalone macaddr hash test, but that can also happen in a followup

Reply:
Oh, sorry, what I had in mind was that can just add the column to the `alltypes` table inside the test, but a standalone test would also work.

In any case, yes, happy to do it in a follow-up, just because I'm not sure about some of the other behaviors mentioned for `macaddr` (and also because I think that's a much less important use case). 😅